samples/bluetooth/direct_test_mode/remote_hci/src/main.c

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/init.h>
+#include <zephyr/net/buf.h>
+#include <zephyr/logging/log.h>
+#include <nrf_rpc/nrf_rpc_ipc.h>
+#include <nrf_rpc_cbor.h>
+
+#include "../../src/transport/hci_uart.h"
+#include "../../rpc/dtm_serialization.h"
+
+LOG_MODULE_REGISTER(serialization_layer, CONFIG_DTM_REMOTE_HCI_LOG_LEVEL);
+
+NRF_RPC_IPC_TRANSPORT(hci_group_tr, DEVICE_DT_GET(DT_NODELABEL(ipc0)), "nrf_rpc_ept");
+NRF_RPC_GROUP_DEFINE(hci_group, "hci_remote", &hci_group_tr, NULL, NULL, NULL);
+
+static K_FIFO_DEFINE(dtm_put_queue);
+
+static void dtm_hci_put_wrapper(struct net_buf *buf);
+
+static void rsp_error_code_send(const struct nrf_rpc_group *group, int err_code)
+{
+	struct nrf_rpc_cbor_ctx ctx;
+
+	NRF_RPC_CBOR_ALLOC(group, ctx, CBOR_CTX_SIZE);
```

Comment:
Because the messages transferred can be big and I currently have no way to test if smaller buffer can be successful in all scenarios. Probably ~500 bytes would be ok, but in this sample nothing else is using ipc, so overshooting is acceptable in my opinion.